app.js

```diff
@@ -18,10 +18,10 @@ app.set('view engine', 'ejs');
 app.use(logger('dev'));
 app.use('/static', express.static(path.join(__dirname, 'public')));
 
-
-app.get("/sample", function(req,res){
-	// here's a sample route
-})
+// use route to index
+app.use('/', require('./routes/index'));
+// use route to pizza
+app.use('/pizza', require('./routes/pizza'));
```

Comment:
I like how these are defined, however in ./routes/index.js you have a route that looks like it belongs in ./routes/pizza.js. I am not sure what the rationale for including index.js at all is, but I think it was meant only to include the home page.